css-two-column-layout/styles.css

```diff
@@ -0,0 +1,51 @@
+.container {
+  width: 70%;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.head-font-property {
+  font-weight: 700;
+  font-size: 40px;
+}
+
+.subhead-font-property {
+  margin-top: 40px;
+}
+
+.font-property {
+  font-family: 'Karla', sans-serif;
+}
+
+.column-one {
```

Comment:
I would make this `column-full` to make it clear what the size is. I get that you mean there will be only one column. in the row, but it could be easily mistaken as a count. 

Also, you should relocate the `text-align` property to a separate class you can add along with this, `column` classes should only ever hold a `width` or `flex-basis` property